testsuite/integration-arquillian/pom.xml

```diff
@@ -363,7 +398,7 @@
             </build>
         </profile>
         <profile>
-            <id>db-mysql</id>
+            <id>db-mysql57</id>
```

Comment:
https://github.com/keycloak/keycloak/blob/master/docs/tests-db.md#using-built-in-profiles-to-run-database-tests-using-docker-containers needs to be updated as well

Reply:
Fixed, thanks!